StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Extensions/Intent+Link.swift

```diff
@@ -60,6 +60,17 @@ extension Intent {
         }
     }
 
+    var disableLinkSignup: Bool {
+        switch self {
+        case .paymentIntent(let paymentIntent, _):
+            return paymentIntent.linkSettings?.disableSignup ?? false
+        case .setupIntent(let setupIntent, _):
+            return setupIntent.linkSettings?.disableSignup ?? false
+        case .deferredIntent(let elementsSession, _):
+            return elementsSession.linkSettings?.disableSignup ?? false
```

Comment:
Maybe not a big deal, but if this flag ever goes away, should we default to `false` or `true` 🤔